changelogs/fragments/acd-content-dir.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- ACD - Add content dir where Ansible will look for content provided by ACD
```

Comment:
Remove this fragment. The first fragment already provides a good description of the changes.

Reply:
Oh, I missed that sivel added this earlier